text/0000-pose-graph-model.md

```diff
@@ -0,0 +1,118 @@
+# Pose Graph Refactoring 1: Pose Graph Model
```

Comment:
Feel free to upload the next work-in-progress RFCs when you have drafts so it's easier to see the big picture.

Reply:
Sure, I will upload soon the `PoseGraphController` part.